testkit/src/main/resources/postgresql/test/migration/V1__test_db_baseline.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE test_table (pk TEXT NOT NULL,data TEXT NOT NULL,constraint pk_pk primary key(pk));
```

Comment:
I don't think we need migrations for this do we? This is strictly for test cases so I think we can just call create the table directly with Slick using `DbManagement.createTable()`?

Reply:
I think this is preferred, this way we can test that the migrations are correctly hooked up to the tests

Reply:
This is just test code isn't it? We don't need migrations for test code IMO